faculty_cli/cli.py

```diff
@@ -1451,3 +1450,147 @@ def dataset_ls(project, prefix, show_hidden):
         prefix, project_id=project_id, show_hidden=show_hidden
     ):
         click.echo(item)
+
+
+@cli.group()
+def template():
+    """Manipulate templates in the Faculty knowledge centre."""
+    pass
+
+
+@template.command()
+def init():
+    """Create a blank template."""
+    print("hello")
+
+
+@template.command()
+@click.argument(
+    "source_template_name"
+)  # Outstanding question as to whether this should also alllow the template_ID
+@click.option(
+    "--version",
+    default=None,
+    help="The specific version of the template to use.",
+)
+@click.option(
+    "--directory",
+    default=Path.cwd(),
+    help="The optional target directory where the template will be placed.",
+    type=click.Path(),
```

Comment:
```suggestion
    type=click.Path(exists=False, file_okay=True),
```

Reply:
These are the default values. Do you think we need the add them explicitly?

Reply:
Oops, I meant them to be the other way round, i.e. get click to check that it's a directory that already exists (`exists=True, file_okay=False`). However, I've since gone back on the idea that we should assert that it already exists - this would be inconsistent with how `git clone` works, which we're sort-of mirroring. I suggest keeping this `type` argument as is.

Reply:
👍